app/keepers/keepers.go

```diff
@@ -541,6 +546,16 @@ func (ak *AppKeepers) InitKeepers(
 		appparams.AccGov.String(),
 	)
 
+	ak.PFMRouterKeeper = pfmrouterkeeper.NewKeeper(
+		appCodec,
+		ak.keys[pfmroutertypes.StoreKey],
+		ak.TransferKeeper,
+		ak.IBCKeeper.ChannelKeeper,
+		ak.BankKeeper,
+		ak.IBCKeeper.ChannelKeeper, // TODO replace ICS4Wrapper with RateLimitKeeper when implemented
```

Comment:
for send path to be `for send path to be: SendPacket -> Transfer -> Callbacks -> PFM -> Fee -> IBC core (ICS4Wrapper)` 

shouldn't this acutally be FeeKeeper ?

Reply:
updated! thanks